site/en/gemma/docs/paligemma/inference-with-keras.ipynb

```diff
@@ -200,7 +200,7 @@
       },
       "outputs": [],
       "source": [
-        "!pip install -U -q keras-nlp kagglehub"
+        "!pip install -U -q keras-hub kagglehub"
```

Comment:
This creates a warning when run in Colab:

```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
keras-nlp 0.18.1 requires keras-hub==0.18.1, but you have keras-hub 0.20.0 which is incompatible.
```

Since Colab has keras-nlp installed already, and the issue is that the version is too old, maybe keep both `keras-nlp` and `keras-hub` in the install command? You might be able to use `--upgrade-strategy=eager` but that is probably overkill.

Reply:
Added NLP as well, it should work without the warning now.